extensions/wikia/AdEngine/templates/recovered_message.mustache

```diff
@@ -0,0 +1,9 @@
+<div class="recovered-message {{positionClass}}">
+    <img src="/skins/oasis/images/recovered_message_icon.png" class="icon">
+    <div class="message">
+        <div class="dialog-pointer"></div>
+        <h3>{{{header}}}</h3>
```

Comment:
I don't think we have to unescape it. The `{{{text}}}` will have HTML but here we can make it more safe and escape it as usual with `{{` instead of `{{{`.